app/src/main/java/org/oppia/app/topic/practice/TopicPracticeFragmentPresenter.kt

```diff
@@ -35,16 +35,16 @@ class TopicPracticeFragmentPresenter @Inject constructor(
 ) : SubtopicSelector {
```

Comment:
Can we remove these 2 unused variables on line number 32 and 33?
```
  private val analyticsController: AnalyticsController,
  private val oppiaClock: OppiaClock,
```

Reply:
Removed.